sklearn/utils/multiclass.py

```diff
@@ -137,8 +137,20 @@ def is_multilabel(y):
     >>> is_multilabel(np.array([[1, 0, 0]]))
     True
     """
+    import warnings
```

Comment:
We should import warnings on the top of the file

Reply:
Ok